src/chapters/errors-and-debugging/error-types.rst

```diff
@@ -0,0 +1,93 @@
+Error Types
+===========
+
+.. index::
+   single: error; syntax
+   single: error; type
+   single: error; runtime
+   single: object; built-in
+
+An **error type** is the classification that JavaScript uses internally to group errors based on their cause. In future lessons, we will come to understand that an error type is actually something called a **built-in object**. For now, however, understanding the different types of errors will help us become faster at debugging.
+
+Each error that JavaScript reports has an error type, and the type is included in the error message. For example, one of the messages we saw previously reported the error type as ``SyntaxError``.
+
+::
+
+   /Users/chris/dev/sandbox/js/syntax.js:2
+   console.log("Hello, name);
+               ^^^^^^^^^^^^^^
+
+   SyntaxError: Invalid or unexpected token
+      at new Script (vm.js:85:7)
+      at createScript (vm.js:266:10)
+      at Object.runInThisContext (vm.js:314:10)
+      at Module._compile (internal/modules/cjs/loader.js:698:28)
+      at Object.Module._extensions..js (internal/modules/cjs/loader.js:749:10)
+      at Module.load (internal/modules/cjs/loader.js:630:32)
+      at tryModuleLoad (internal/modules/cjs/loader.js:570:12)
+      at Function.Module._load (internal/modules/cjs/loader.js:562:3)
+      at Function.Module.runMain (internal/modules/cjs/loader.js:801:12)
+      at internal/main/run_main_module.js:21:11
+
+We have now seen two error types, ``ReferenceError`` and ``SyntaxError``. There are several other `error types in JavaScript <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects#Fundamental_objects>`_, such as ``TypeError`` and ``RangeError``. 
+
+The following table describes all JavaScript error types. Some of these relate to coding concepts we have not covered yet, but we include them here as a reference for future use.
+
+.. list-table:: JavaScript Error Types
```

Comment:
Remove the table title until we change the styling.

Reply:
I'm going to leave these titles. It will be much simpler to do so than to go back and re-label all of these tables.